CONTRIBUTING.md

```diff
@@ -15,6 +15,17 @@ Development happens against the `master` branch following the [GitHub flow model
 
 Pull requests against `master` trigger automated tests that are run through Azure DevOps. Additional test suites are run periodically. When adding new code paths or features, tests are a requirement to complete a pull request. They should be added in the `test` directory.
 
+To build the fairlearn dashboard after making changes to it, [install Yarn](https://yarnpkg.com/lang/en/docs/install), and then do the following:
+
+```bash
+cd fairlearn/widget/js
+yarn install
+yarn build:all
+rm -rf dist
+rm -rf lib
+rm -rf node_modules
```

Comment:
alternatively you could put these in a makefile and tell users of the make command

Reply:
Good idea. Isn't makefile something platform specific, though? We've seen this many times in the past that there are people who write powershell scripts, others write bash scripts, etc. and others have to rewrite it for their OS or set of tools. The only good answer to this problem that I've seen is to write everything in python since it runs cross-platform.

Reply:
Ah, indeed I wasn't far from the truth! This already exists as a powershell file under scripts. I guess I don't need this description at all then and can just refer to the script. Unfortunately powershell won't help people other than Windows users :-(

Reply:
Turns out that isn't quite true :O https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-linux?view=powershell-7
Pretty sure most people wouldn't go that route though :D

Reply:
Powershell is available on Linux, and ADO does some sort of magic to make scripts like this run (well, PowerShell itself aliases `ls`, `rm` and `cd` to its own internal commands). And I believe `make` is able to drive people mad on multiple platforms too ;-) It's all a question of what's likely to be installed by default.

Reply:
I added a link to the script that does this.